core/src/main/java/com/novoda/landingstrip/ScrollOffsetCalculator.java

```diff
@@ -1,26 +1,29 @@
 package com.novoda.landingstrip;
 
 import android.view.View;
+import android.view.ViewGroup;
 
 class ScrollOffsetCalculator {
 
     private static final float HALF_MULTIPLIER = 0.5f;
     private static final float ROUNDING_OFFSET = 0.5f;
 
-    private final TabsContainer tabsContainer;
+    private final View rootView;
+    private final ViewGroup tabsContainer;
 
-    ScrollOffsetCalculator(TabsContainer tabsContainer) {
+    ScrollOffsetCalculator(View rootView, TabsContainerView tabsContainer) {
+        this.rootView = rootView;
         this.tabsContainer = tabsContainer;
     }
 
     int calculateScrollOffset(int position, float pagerOffset) {
-        if (!tabsContainer.hasTabAt(position)) {
+        if (!hasTabAt(position)) {
             return 0;
         } else {
-            View tabForPosition = tabsContainer.getTabAt(position);
+            View tabForPosition = tabsContainer.getChildAt(position);
 
-            float tabStartX = tabForPosition.getLeft() + getHorizontalScrollOffset(tabForPosition, pagerOffset);
-            float viewMiddleOffset = getTabParentWidth() * HALF_MULTIPLIER;
+            float tabStartX = tabForPosition.getLeft() + getHorizontalScrollOffset(position, pagerOffset);
```

Comment:
Why this change? You're doing `tabForPosition = tabsContainer.getChildAt(position)` twice now. Not a big perf concern, I just don't follow the reasoning